cogs/downloader.py

```diff
@@ -508,6 +508,8 @@ def get_dir_name(self, url):
         return git_name[:-4]
 
     def is_lib_installed(self, name):
+        if "git+" in name:
```

Comment:
Would suggest changing it to
`if` name.startswith("git+"):`

But otherwise its fine